cordova-lib/spec-cordova/HooksRunner.spec.js

```diff
@@ -249,7 +249,7 @@ describe('HooksRunner', function() {
                 var hooksOrderFile = path.join(projectRoot, 'hooks_order.txt');
 
                 return hooksRunner.fire(test_event, hookOptions).then(function() {
-                    expect(hooksOrderFile).toExist();
+                    expect(hooksOrderFile).toBeDefined();
```

Comment:
Oh okay I can change it back... for some reason when I take out toExist, the error "cannot read property of apply" goes away. 
(from helpers.js) ... changing this.addMatchers to jasmine.addMatchers, should allow toExist to function properly?
// Add the toExist matcher.
beforeEach(function () {
    jasmine.addMatchers({
        'toExist': function () {
            return {
                compare: function (testPath) {

                    var result = {};
                    result.pass = fs.existsSync(testPath);

                    if(result.pass) {
                        result.message = 'Expected file ' + testPath + ' to exist.';
                    } else {
                        result.message = 'Expected file ' + testPath + ' to not exist.';
                    }

                    return result

                }
            }
        }
    });
});